torchci/lib/types.ts

```diff
@@ -133,8 +135,10 @@ function formatHudURL(
     params.repoName
   }/${encodeURIComponent(params.branch)}/${params.page}`;
 
+  base += `?per_page=${params.per_page}`;
```

Comment:
nit: you can probably use the javascript url to add query params but I don't think we do that anywhere else so this is fine I guess.